packages/strapi-connector-bookshelf/lib/queries.js

```diff
@@ -176,36 +176,35 @@ module.exports = function createQueryBuilder({ model, modelKey, strapi }) {
   }
 
   function search(params, populate) {
-    // Convert `params` object to filters compatible with Bookshelf.
-    const filters = modelUtils.convertParams(modelKey, params);
+    const filterParams = _.clone(params);
+
+    _.unset(filterParams, '_q');
+
+    const filters = convertRestQueryParams(filterParams);
 
     return model
       .query(qb => {
         buildSearchQuery(qb, model, params);
-
-        if (filters.sort) {
-          qb.orderBy(filters.sort.key, filters.sort.order);
-        }
-
-        if (filters.start) {
-          qb.offset(_.toNumber(filters.start));
-        }
-
-        if (filters.limit) {
-          qb.limit(_.toNumber(filters.limit));
-        }
       })
+      .query(buildQuery({ model, filters }))
       .fetchAll({
         withRelated: populate,
       })
       .then(results => results.toJSON());
   }
 
   function countSearch(params) {
+    const filterParams = _.clone(params);
+
+    _.unset(filterParams, '_q');
+
+    const filters = convertRestQueryParams(filterParams);
+
     return model
       .query(qb => {
         buildSearchQuery(qb, model, params);
```

Comment:
can you refactor buildSearchQuery to be consistent with buildQuery :) (e.g `bSQ = ({ model, params }) => qb => {}`)